Debugging/COSQLiteStore+Graphviz.m

```diff
@@ -179,11 +179,9 @@ void COViewDOTGraphFile(NSString *dotFilePath)
             continue;
         }
 
-        // NOTE: Using NSTask rather than system() breaks 'po [objectGraphContext showGraph]' in LLDB on 10.7 
-        system([[NSString stringWithFormat: @"%@ -Tpdf '%@' -o '%@'",
-                                            executablePath,
-                                            dotFilePath,
-                                            pdfPath] UTF8String]);
+        NSTask *task = [NSTask launchedTaskWithLaunchPath:executablePath
```

Comment:
A space is missing between launchedTaskWithLaunchPath: and executablePath. Same right after arguments:.